src/index.ts

```diff
@@ -51,27 +52,46 @@ function createClass(bundle: AnserJsonEntry): string | null {
   return classNames;
 }
 
-interface Colors {
-  color?: string;
-  backgroundColor?: string;
-}
-
 /**
  * Create the style attribute.
  * @name createStyle
  * @function
  * @param {AnserJsonEntry} bundle
  * @return {Object} returns the style object
  */
-function createStyle(bundle: AnserJsonEntry): Colors {
-  const style: Colors = {};
+function createStyle(bundle: AnserJsonEntry): React.CSSProperties {
+  const style: React.CSSProperties = {};
   if (bundle.bg) {
     style.backgroundColor = `rgb(${bundle.bg})`;
   }
   if (bundle.fg) {
     style.color = `rgb(${bundle.fg})`;
   }
-
+  switch (bundle.decoration) {
```

Comment:
Might want to rebase now that the other PR has been merged.

Reply:
rebased! :)